src/main/java/org/gridsuite/modification/server/modifications/EquipmentDeletion.java

```diff
@@ -36,7 +44,17 @@ public void apply(Network network, Reporter subReporter) {
         if (identifiable instanceof Connectable) {
             new RemoveFeederBay(modificationInfos.getEquipmentId()).apply(network, true, subReporter);
         } else if (identifiable instanceof HvdcLine) {
-            ((HvdcLine) identifiable).remove();
+            List<String> shuntCompensatorIds = Stream.concat(
+                    modificationInfos.getMcsOnSide1() != null ? modificationInfos.getMcsOnSide1().stream() : Stream.of(),
+                    modificationInfos.getMcsOnSide2() != null ? modificationInfos.getMcsOnSide2().stream() : Stream.of())
+                .filter(ShuntCompensatorSelectionInfos::isSelected)
```

Comment:
Report a warning for shunt compensators not found and eliminate them in the ids given to powsybl algo ...

Reply:
yes done